Wallofcoin Buying Wizard

sujalBandhara opened the pull request:
Added Wall of Coin Buying Wizard [Mainnet]
[wallofcoins website](https://wallofcoins.com/en/)

sujalBandhara committed:
Wallofcoin Buying Wizard

Added Wall of Coin Buying Wizard

furszy commented:
First comment: Fuck, 17k changes?. :P

Will review it in parts.

furszy commented on app/build.gradle:
```diff
@@ -87,4 +96,20 @@ dependencies {
+    compile 'com.squareup.okhttp3:logging-interceptor:3.3.1'
+    compile 'com.squareup.okhttp:logging-interceptor:2.7.5'
+    compile 'com.google.code.findbugs:jsr305:2.0.3'
+    compile 'org.dashj:dashj-core:0.14.4.3'
```
Why are you using dashj and not the pivxj that is already in the project? This will cause a lot of redundant dependencies

jaydeep82 replied:
We have removed dependency of 'org.dashj:dashj-core:0.14.4.3' now.

https://github.com/PIVX-Project/PIVX-Android/pull/8/commits/359d68e08713226fd43ee5d46ade5cf55b767ca7

furszy commented on app/src/main/java/pivx/org/pivxwallet/ui/base/BaseRecyclerFragment.java:
```diff
@@ -75,6 +77,21 @@ public void onRefresh() {
                 }
         );
+
+        imgEmptyView.setOnClickListener(new View.OnClickListener() {
```
This class is called BaseRecyclerFragment, don't add specific logic here. Move your logic down in the class hierarchy.

jaydeep82 replied:
We have removed logic "imgEmptyView.setOnClickListener" from BaseRecyclerFragment now.

https://github.com/PIVX-Project/PIVX-Android/pull/8/commits/359d68e08713226fd43ee5d46ade5cf55b767ca7

sandeeppatel1986 committed:
Clean BaseRecyclerFragment & Remove dependancy of 'org.dashj:dashj-core:

1) Move logic  "imgEmptyView.setOnClickListener " from BaseRecyclerFragment
2) Remove dependancy of 'org.dashj:dashj-core:0.14.4.3' from gradle.

sandeeppatel1986 committed:
Removed intermediate files

Removed intermediate files & unused files

rgenito commented:
@furszy the good news is that the bulk of these changes are UI-building elements :)
